sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -235,6 +241,8 @@ export class ServiceBusClient {
    * @param {ServiceBusClientOptions} options - Options to control ways to interact with the
    * Service Bus Namespace.
    * @returns {ServiceBusClient}
+   * @throws {Error} Thrown if the connection string contents were malformed.
```

Comment:
`createFromAadTokenCredentials` does not take connection string

Reply:
For this and above, maybe if the host name passed is not as expected?

Reply:
I dont believe we make any kind of checks on the host name or its contents.

Reply:
if hostname has `;` in it, then the parseConnectionString would attempt to split things and fail if it doesn't find a `=` within the host name

Reply:
That would still fall under the category of the program failing right at the start and therefore the same category where say a string is passed to `peek()` operation and we get the TypeError. As discussed elsewhere, we dont intend to document such errors.

Both `createFromTokenProvider` and `createFromAadTokenCredentials` do no take connection string and having documentation that refers to a "connection string" is misleading.

In conclusion, we can skip updating the docs for `createFromConnectionString`, `createFromTokenProvider` and `createFromAadTokenCredentials`

Reply:
updated